Bug: NPEx when using serverId

on my POM I have this 
```
	<build>
...
		<plugins>
			<plugin>
				<groupId>com.spotify</groupId>
				<artifactId>docker-maven-plugin</artifactId>
				<version>0.2.8</version>
					<configuration>
						<serverId>docker-hub</serverId>
...
					</configuration>
			</plugin>
		</plugins>
	</build>
```
under my settings.xml  (xpath `//settings/servers/server`) I have this:
```
    <server>
    	<id>docker-hub</id>
    	<username>******</username>
    	<password>*******</password>
    </server>
```

when running `mvn clean install docker:build` I get this errors

```
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.2.8:build (default-cli) on project k2: Exception caught: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.spotify:docker-maven-plugin:0.2.8:build (default-cli) on project k2: Exception caught
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:108)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:76)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:116)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:361)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:213)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException: Exception caught
	at com.spotify.docker.AbstractDockerMojo.execute(AbstractDockerMojo.java:117)
	at com.spotify.docker.BuildMojo.execute(BuildMojo.java:79)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:133)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 19 more
Caused by: java.lang.NullPointerException
	at com.spotify.docker.AbstractDockerMojo.getEmail(AbstractDockerMojo.java:155)
	at com.spotify.docker.AbstractDockerMojo.execute(AbstractDockerMojo.java:86)
	... 22 more
```

--- a/src/main/java/com/spotify/docker/AbstractDockerMojo.java
+++ b/src/main/java/com/spotify/docker/AbstractDockerMojo.java
@@ -152,9 +152,11 @@ abstract class AbstractDockerMojo extends AbstractMojo {
     String email = null;
 
     final Xpp3Dom configuration = (Xpp3Dom) server.getConfiguration();
-    final Xpp3Dom emailNode = configuration.getChild("email");
-    if (emailNode != null) {
-      email = emailNode.getValue();
+    if (configuration != null) {
+      final Xpp3Dom emailNode = configuration.getChild("email");
+      if (emailNode != null) {
+        email = emailNode.getValue();
+      }
     }
 
     return email;

